src/auth/rest.rs

```diff
@@ -128,7 +128,7 @@ impl Authenticator<DefaultUser> for RestAuthenticator {
         let selector = self.selector.clone();
         let regex = self.regex.clone();
 
-        debug!("{} {}", url, body);
```

Comment:
Disabled cause its a bit annoying passing a logger in here. If really want to log here I can make a plan.